app/imports/containers/applyContainer.jsx

```
@@ -7,28 +9,34 @@ import { ApplicationSections } from '/imports/collections/applicationSections.js
 
 import { createApplication } from '/imports/api/methods.js';
 
-const composer = ( props, onData ) => {
+const composer = (props, onData) => {
 
-  const subscription = Meteor.subscribe( 'application.single' );
+  const subscription = Meteor.subscribe('application.single');
 
   // Check if subscription is ready
   if ( subscription.ready() ) {
 
-    // Set some data 
+    // Set some data
     const user = !!Meteor.user() ? Meteor.user() : null;
 
     // Wait for user subscription (kinda)
-    if( user ) {
+    if (user) {
 
       // Look for user's application
       let application = Applications.findOne({ userId: user._id });
 
-      if ( !!application ) {
+      if (!!application) {
+
+        // Check if time is after deadline
+        if (moment() > moment(Meteor.settings.public.applicationDeadline, Meteor.settings.public.dateFormat)) {
```

I think we can fail faster, even before the subscription is ready.